pkgs/development/python-modules/pyfuse3/default.nix

```diff
@@ -1,20 +1,45 @@
-{ stdenv, lib, buildPythonPackage, fetchPypi, pkg-config, fuse3, trio, pytestCheckHook, pytest-trio, which }:
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, cython
+, pkg-config
+, fuse3
+, trio
+, python
+, pytestCheckHook
+, pytest-trio
+, which
+}:
 
 buildPythonPackage rec {
   pname = "pyfuse3";
   version = "3.2.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "22d146dac59a8429115e9a93317975ea54b35e0278044a94d3fac5b4ad5f7e33";
+  disabled = pythonOlder "3.5";
```

Comment:
I've just set this to disabled on `pythonAtLeast "3.10"`, because that seemed reasonable when looking at hydra. So you'll need to rebase this.